commonTestResources/exampleOpenApiFiles/valid/basePathVariations/basePathDefined.yml

```diff
@@ -0,0 +1,14 @@
+swagger: '2.0'
+info:
+  description: Has one path.
+  title: Example OpenApi 2 spec with basePath
+  version: 0.1.0
+basePath: /test
+paths:
+  /responseBody/string:
+    get:
+      responses:
+        200:
+          description: Response body should be a string
+          schema:
+            type: string
```

Comment:
nit: did you run prettier (`yarn format` / `npm run format`)? If yes, I will remember to update our prettier config to add new lines at end of files

Reply:
I did not run prettier, so I will make sure to do that for my next commits.

On a somewhat related note, you should probably update your [CONTRIBUTING](https://github.com/RuntimeTools/OpenAPIValidators/blob/master/CONTRIBUTING.md) documentation. A lot of the commands there are outdated (e.g., `npm run test:coverage` is no longer defined in `package.json`). You may also want to include a few words (or links) about mutation testing, because I don't think it's a very commonly used tool. 

I'm a big fan of this package, and I think you do a really great job managing it! Just want to make sure you keep your contributing guidelines up to date so that you don't have to spend any more time managing this repo than you already do!

Reply:
- automate prettier https://github.com/RuntimeTools/OpenAPIValidators/pull/169
- update contributing.md https://github.com/RuntimeTools/OpenAPIValidators/pull/175